Swift Vocabulary/Swift Vocabulary/ViewControllers/WordsTableViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  WordsTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Dojo on 5/20/20.
+//  Copyright © 2020 Dojo. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+    
+    var vocabWords: [VocabularyWord] = [VocabularyWord(word:"Dude", definition:"A person that is awesome"), VocabularyWord(word:"Sweet", definition:"Something you say when something is awesome")]
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+        
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem
+    }
+    
+    // MARK: - Table view data source
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return vocabWords.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "WordCell", for: indexPath)
+        
+        let word = vocabWords[indexPath.row]
+        
+        cell.textLabel?.text = word.word
+        
+        return cell
+    }
+    
+    
+    
+    // MARK: - Navigation
+
+  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+        if segue.identifier == "ShowDefinitionSegue" {
+            guard let indexPath = tableView.indexPathForSelectedRow else { return }
+
+            let word = vocabWords[indexPath.row]
+
+            let definitionVC = segue.destination as? DefinitionViewController
+            definitionVC?.word = word
+        }
+
+    }
```

Comment:
Nice job unwrapping and casting here. I would maybe suggest unwrapping the destination as well (I know it wasn't demonstrated this way). That would look like this:

``` swift
override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
    // Get the new view controller using segue.destination.
    // Pass the selected object to the new view controller.
    if segue.identifier == "ShowDefinitionSegue" {
        guard let indexPath = tableView.indexPathForSelectedRow,
              let definitionVC = segue.destination as? DefinitionViewController else { return }
        
        let word = vocabWords[indexPath.row]
        
        definitionVC.word = word
    }
    
}
```
This way you avoid having to use optional chaining.